app/src/main/res/values-night/styles.xml

```diff
@@ -1,7 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Your `tool_sheet_view.xml` is still registering as a `binary file not shown` in github. Please find a way to fix

Reply:
![image](https://user-images.githubusercontent.com/112203610/197295975-5cdbd262-cb7a-4cfd-866a-4de3a4d9218b.png)

Reply:
no clue what was going on with this. I ended up just deleting the file and making an identical one and pasting the same xml into it and it would parse. 

re-ran it three times